src/Snppts/Snippets/MauiMsDosUI.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Snppts.Authors;
+using Snppts.Infrastructure;
+using System.Collections.Generic;
+using System;
+using Snppts.Extensions;
+
+namespace Snppts.Snippets
+{
+    public class MauiMsDosUI : IAmASnippet
+    {
+        public string Slug => "maui-ms-dos";
+        public string Title => "Maui MS Dos";
+        public string GithubRepoName => "jm-parent/MSDos4Ever";
```

Comment:
Doesn't seem to exist? Is it still set to private?

Reply:
Project set to public ✅